kubernetes/README.md

```diff
@@ -0,0 +1,26 @@
+# Creating a Kubernetes Deployment
+
+## Instructions
+
+Copy your private key which you have configured with JIRA into the
+current directory, and name it `jira_privatekey.pem`.
```

Comment:
Why does it have to be named this way? The user needs to make this file's content the value of a secret's key.